### PR TITLE
Create specific umd file for woocommerce

### PR DESCRIPTION
**Pull request by FranceBe**
A merchant saw an issue with our widget in WooCommerce module 
https://patchstack.com/database/vulnerability/alma-gateway-for-woocommerce/wordpress-alma-plugin-5-1-3-cross-site-scripting-xss-vulnerability?_a_id=110

So this PR is meant to fix the issue only for WooCommerce as we do not wish to introduce too many changes for the rest of the widgets usage.

Resources : 
- https://www.freecodecamp.org/news/innerhtml-vs-innertext-vs-textcontent/
- https://cheatsheetseries.owasp.org/cheatsheets/DOM_based_XSS_Prevention_Cheat_Sheet.html#rule-7-fixing-dom-cross-site-scripting-vulnerabilities

### CODE REVIEW

File: scripts/umd-for-woocommerce.js [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+// For WooCommerce, we need to replace innerHTML with innerText
+// to avoid XSS issues warnings
+// https://developer.mozilla.org/en-US/docs/Web/API/Element/innerHTML#security_considerations
+// In our widget, innerHTML is introduced by the library `react-dom` and `react` that are included
+// into the `widgets.umd.js` file as our widget is built with React.
+// We can not remove `react-dom` and `react` as they are required for our widget to work.
+// We can not change the node_modules nor modify how they are included in our library.
+// We need a workaround to replace innerHTML with innerText in the `widgets.umd.js` file for WooCommerce integration.
+// See this example as reference https://github.com/jtagcat/simple-tab-groups/blob/1d358d31bdd736fdb7fbe196450636e09359c8e5/addon/scripts/remove-evals.js#L43
+const replaceInnerHtml = () => {
```
Comment (MarcAntoineGautier) [inline scripts/umd-for-woocommerce.js]: What happen if the file does not exist? For example, if the build command fails or if the script is executed elsewhere?

File: scripts/umd-for-woocommerce.js [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+  const fileContent = readFileSync(fileName, 'utf8')
+  const fileContentForWC = fileContent.replace(/innerHTML/g, 'innerText')
```
Comment (MarcAntoineGautier) [inline scripts/umd-for-woocommerce.js]: Are we sure the code still works if you replace all the innerHTML like this? Maybe some occurrences need another element to be replaced with. 
Maybe more info [here](https://developer.mozilla.org/en-US/docs/Web/API/Element/innerHTML)

Comment (FranceBe) [reply scripts/umd-for-woocommerce.js]: Seems okay for now 😬

**Review by MarcAntoineGautier — APPROVED** [review]
Not tested in real life but the script looks good to me

**Pull request merged**